idem_azurerm/states/azurerm/containerinstance/group.py

```diff
@@ -0,0 +1,546 @@
+
+        **secure_value**: The value of the secure environment variable.
+    - **volume_mounts**: A list of volume mounts available to the container instance.
+        **name**: Required if volume_mounts is used. The name of the volume mount.
+
+        **mount_path**: Required if volume_mounts is used. The path within the container where the volume should
+        be mounted. Must not contain colon (:).
+
+        **read_only**: Boolean flag indicating whether the volume mount is read-only.
+    - **liveness_probe**:
+            **exec_property**:
+                **command**: The commands to execute within the container.
+            **http_get**:
+                **path**: The path to probe.
+
+                **port**: Required if http_get is used. The port number to probe.
+
+                **scheme**: The scheme. Possible values include: 'http', 'https'
+            **initial_delay_seconds**: The initial delay seconds.
+
+            **period_seconds**: The period seconds.
+
+            **failure_threshold**: The failure threshold.
+
+            **success_threshold**: The success threshold.
+
+            **timeout_seconds**: The timeout seconds.
+    - **readiness_probe**:
+            **exec_property**:
+                **command**: The commands to execute within the container.
+            **http_get**:
+                **path**: The path to probe.
+
+                **port**: Required if http_get is used. The port number to probe.
+
+                **scheme**: The scheme. Possible values include: 'http', 'https'
+            **initial_delay_seconds**: The initial delay seconds.
+
+            **period_seconds**: The period seconds.
+
+            **failure_threshold**: The failure threshold.
+
+            **success_threshold**: The success threshold.
+
+            **timeout_seconds**: The timeout seconds.
+
+    :param os_type: The operating system type required by the containers in the container group. Possible values
+        include: 'Windows', 'Linux'
+
+    :param restart_policy: Restart policy for all containers within the container group. Possible values are:
+    - ``Always``: Always restart
+    - ``OnFailure``: Restart on failure
+    - ``Never``: Never restart.
+
+    :param identity: A dictionary defining a ContainerGroupIdentity object which represents the identity for the
+        container group.
+
+    :param image_registry_credentials: A list of dictionaries defining ImageRegistryCredential objects for the image
+        registry credentials.
+
+    :param ip_address: A dictionary defining an IpAddress object which represents the IP address for the container
+        group. Possible keys are:
+    - ``ports``: Required if ip_address is used. The list of ports exposed on the container group.
+    - ``type``: Required if ip_address is used. Specifies if the IP is exposed to the public internet or private VNET.
+    Possible values include: 'Public', 'Private'
```

Comment:
I would indent this more so you can more easily tell that this sentence is still a part of the `type` parameter description.